R/MzTabReader.R

```diff
@@ -400,8 +456,12 @@ RTUnitCorrection = function(dt)
 {
   "Convert all RT columns from seconds (OpenMS default) to minutes (MaxQuant default)"
   
-  cn_rt = grepv("retention.time|retention.length", names(dt))
-  dt[, c(cn_rt) := lapply(.SD, function(x)  x / 60), .SDcols = cn_rt]
+  #retention.time is mandatory for mzTab
```

Comment:
maybe add a comment:
`# heuristic to detect presence of unit:seconds; if retention.time has is, we assume that all rt-columns are in seconds`